### PR TITLE
AddAndDeleteWaitersTest

### PR DESCRIPTION
**Pull request by Zeicks**

### CODE REVIEW

File: src/main/java/com/easyrest/components/owner/CreateNewEmployee.java [hunk 1]
```diff
@@ -50,4 +50,11 @@ public void clickOnButtonAdd() {
     public void clickOnButtonCancel() {
         driver.findElement(cancelButton).click();
     }
+
+    private boolean isDisplayed(By element) {
```
Comment (martadiachok) [inline src/main/java/com/easyrest/components/owner/CreateNewEmployee.java]: what this method returns?

File: src/test/java/com/easyrest/tests/owner/ActionWithWaitersTest.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+
+    @Test
+    public void deleteWaiter() {
+        signInPage = new SignInPage(driver);
+        signInFacade = new SignInFacade(driver);
+        createNewWaiterFacade = new CreateNewWaiterFacade(driver);
+        waiters = new Waiters(driver);
+        manageMenu = new ManageMenu(driver);
+        leftSideMenu = new LeftSideMenu(driver);
+        signInPage.goToSignInPage();
```
Comment (martadiachok) [inline src/test/java/com/easyrest/tests/owner/ActionWithWaitersTest.java]: maybe should do annotation @BeforeTest with this methods: 
  signInPage.goToSignInPage();
  signInFacade.signIn(ownerEmail, ownerPassword);

File: src/main/java/com/easyrest/components/owner/CreateNewEmployee.java (hunk 1 above)
Comment (Zeicks) [reply src/main/java/com/easyrest/components/owner/CreateNewEmployee.java]: Amount of elements

File: src/test/java/com/easyrest/tests/owner/ActionWithWaitersTest.java (hunk 2 above)
Comment (Zeicks) [reply src/test/java/com/easyrest/tests/owner/ActionWithWaitersTest.java]: Done

File: src/main/java/com/easyrest/components/owner/CreateNewEmployee.java (hunk 1 above)
Comment (dmatsev) [reply src/main/java/com/easyrest/components/owner/CreateNewEmployee.java]: I think the method name doesn't correspond to its implementation, so just rename it with a meaningful name.

**Review by dmatsev — APPROVED** [review]
Approve with comment.

**Review by martadiachok — APPROVED** [review]

**Review by vadymtsehelny — APPROVED** [review]

**Pull request merged**